client/coniksclient/internal/cmd/common.go

```diff
@@ -39,5 +40,5 @@ func loadConfigOrExit(cmd *cobra.Command) *client.Config {
 
 // append "\r\n" to msg and then write to terminal in raw mode.
 func writeLineInRawMode(term *terminal.Terminal, msg string) {
-	term.Write([]byte(msg + "\r\n"))
+	term.Write([]byte("<" + time.Now().Format("15:04:05.999999999") + "> " + msg + "\r\n"))
```

Comment:
I would prefer to have it in the config. But if its easier/more convenient a command like you propose is fine, too.